src/search_command.sh

```diff
@@ -11,8 +11,10 @@ search_repo() {
 
   ## Search directories matching search text
   blue "Matching packages:\n"
-  find "$repo_path" -type d -not -path '*/\.*' | grep --color=always "$text" |
-    sed "s#${repo_path}/#${prefix}#g" | sed 's#/info##'
+  find "$repo_path" -type d |
+    grep -v "$repo_path.*/\." |
+    grep --color=always "$text" |
+    sed "s#${repo_path}/#${prefix}#g"
```

Comment:
Probably works fine for 99% of people, though I'm not _super_ excited about the repo path being treated as a regex. Perhaps slightly more technically correct:

```suggestion
  path_length=$((${#repo_path}+2))
  find "$repo_path" -type d |
    cut --characters "$path_length-" |
    grep -v ".*/\." |
    grep --color=always "$text" |
    sed "s#^#${prefix}#"
```

Reply:
I am not also not crazy about this excessive piping.
Perhaps we should search for dirs that contain `main` files instead of searching for dirs?